src/game/AI/ScriptDevAI/scripts/eastern_kingdoms/karazhan/boss_nightbane.cpp

```diff
@@ -82,12 +82,14 @@ struct boss_nightbaneAI : public npc_escortAI
     uint32 m_uiFireballBarrageTimer;
 
     bool m_bCombatStarted;
+    bool m_bNightbaneDamage;
```

Comment:
Why this variable? If you remove it, I can merge it.

Reply:
It's done.